### DISCUSSION TITLE
Enabling debug logging in community_version.py

### DISCUSSION THREAD

**Discussion by bobomatic** [zero-to-mastery/ascii-art-2021#102] [Q&A] [answered]
Im not familiar with logger. Anyone know how to enable the DEBUG level logging? 

I see there is a verbose_output handler in logger_config.py but I don't know what it does.

Cheers

Bob

**Comment by ndeoligence** [comment, marked as answer]
Thanks for the question @bobomatic - and sorry for the delayed reply (I had a busy evening).

To show debug messages you have to change the handler to DEBUG.

Looking at the project's `logger_config.py` it looks like we only have 2 logger **handlers**. 
1. **logfile** logs to a file (only logs errors, since `level: ERROR`.
2. **verbose_output** logs to stdout (terminal). It's already supposed to show debug messages - since it's got `level: DEBUG,`

I hope it helps. Otherwise you could check out [this resource](https://docs.python.org/3/howto/logging.html#logging-basic-tutorial) for more detailed look at logging config.

**Reply by bobomatic** [reply]
Hi. Thanks for taking the time to reply! OK so stdout is terminal - debug - and that is working fine. 

I couldn't see the error log file in the project directory ".text-ascii-art-errors.log". But now I realise that it is a hidden file!
